WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -1562,6 +1641,32 @@ extension ProductsViewController {
     }
 }
 
+private extension ProductsViewController {
+    func reloadSections() {
+        if favoriteProducts.isNotEmpty {
+            sections = [ProductListViewSection(type: .favourites, products: favoriteProducts),
+                        ProductListViewSection(type: .allProducts, products: resultsController.fetchedObjects)]
+        } else {
+            sections = [ProductListViewSection(type: .allProducts, products: resultsController.fetchedObjects)]
+        }
+    }
+
+    func indexPath(for productID: Int64) -> IndexPath? {
+        guard let allProductsSectionIndex = sections.firstIndex(where: {$0.type == .allProducts }),
```

Comment:
Nit: Can you leave a comment explaining why we're supporting index path only for the All Products section?

Reply:
I have renamed the method in https://github.com/woocommerce/woocommerce-ios/pull/12287/commits/d90870480206b507c732fec59fc00649d77a9c35

This is used to select the product when changes are made in a split view. I don't understand the nuances of the split view part yet. I would prefer to keep the selected product and scroll position untouched until we get time to revisit this work.